packages/sanity/package.json

```diff
@@ -202,6 +202,7 @@
     "@sanity/block-tools": "3.34.0",
     "@sanity/cli": "3.34.0",
     "@sanity/client": "^6.15.5",
+    "@sanity/codegen": "workspace:*",
```

Comment:
> I don't think workspace:* will resolve when publishing this to npm.

It will: https://pnpm.io/workspaces#publishing-workspace-packages

Ideally we should use the workspace protocol for all workspace packages.